script/testing/README.md

```diff
@@ -7,16 +7,17 @@ All tests are compatible with python3
 - `micro_bench`: entry script to run the microbenchmark tests
 - `oltpbench`: entry script to fire an oltp bench test
 - `artifact_stats`: entry script to collect the artifact stats
+- `reporting`: utility scripts for posting test data to Django API and formating JSON payloads
 
 ## Util
 `util` folder contains a list of common Python scripts
-- `common`: functions that can be used in many different settings
-- `constants`: all the constants used in the any file under the `util` or across the different tests
-- `NoisePageServer`: A class that can start, stop, or restart an instance of the DBMS
-- `TestServer`: the base class for running all types of tests
-- `TestCase`: the base class for all types of tests
-- `TestJUnit`: the test class for junit tests
-- `TestOLTPBench`: the test class for oltp bench tests
+- `common.py`: functions that can be used in many different settings
+- `constants.py`: all the constants used in the any file under the `util` or across the different tests
+- `db_server.py`: It provides a `NoisePageServer` class that can start, stop, or restart an instance of the NoisePage
+- `test_server.py`: the base `TestServer` class for running all types of tests
+- `test_case.py`: the base `TestCase` class for all types of test cases.
+- `mem_metrics.py`: the `MemoryMetric` class and `MemoryInfo` named tuple to manage the memory related information during the run time of the tests.
+- `periodic_task.py`: the `PeriodicaTask` class to wrap periodic python subprocess tasks with multithreading.
```

Comment:
I think a little more detail around `PeriodicTask` would help. I understand what this means but I think it might be because I'm familiar with the work. (also check spelling). It might help to phrase it along the lines of `the PeriodicTask class runs a separate thread that will execute a subprocess every x seconds until told to stop.`

Reply:
Yep I think your summary for this is much better than mine.